packages/@react-aria/landmark/stories/Landmark.stories.tsx

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {ActionGroup, Item} from '@react-spectrum/actiongroup';
+import {Cell, Column, Row, TableBody, TableHeader, TableView} from '@react-spectrum/table';
+import {Checkbox} from '@react-spectrum/checkbox';
+import {classNames, useStyleProps} from '@react-spectrum/utils';
+import {Flex} from '@react-spectrum/layout';
+import {Link} from '@react-spectrum/link';
+import {Meta, Story} from '@storybook/react';
+import React, {useRef} from 'react';
+import styles from './index.css';
+import {TextField} from '@react-spectrum/textfield';
+import {useLandmark} from '../';
+
+
+interface StoryProps {}
+
+const meta: Meta<StoryProps> = {
+  title: 'Landmark'
+};
+
+export default meta;
+
+const Template = (): Story<StoryProps> => (props) => <Example {...props} />;
+const NestedTemplate = (): Story<StoryProps> => (props) => <NestedExample {...props} />;
+const TableTemplate = (): Story<StoryProps> => (props) => <TableExample {...props} />;
+const ApplicationTemplate = (): Story<StoryProps> => (props) => <ApplicationExample {...props} />;
+const DuplicateRolesWithLabelsTemplate = (): Story<StoryProps> => (props) => <DuplicateRolesWithLabelsExample {...props} />;
+const DuplicateRolesNoLabelsTemplate = (): Story<StoryProps> => (props) => <DuplicateRolesNoLabelExample {...props} />;
+
+
+function Main(props) {
+  let ref = useRef();
+  let {styleProps} = useStyleProps(props);
+  let {landmarkProps} = useLandmark({...props, role: 'main'}, ref);
+  return <main ref={ref} {...landmarkProps} {...styleProps}>{props.children}</main>;
+}
+
+function Navigation(props) {
+  let ref = useRef();
+  let {styleProps} = useStyleProps(props);
+  let {landmarkProps} = useLandmark({...props, role: 'navigation'}, ref);
+  return <nav ref={ref} {...landmarkProps} {...styleProps}>{props.children}</nav>;
+}
+
+function Region(props) {
+  let ref = useRef();
+  let {styleProps} = useStyleProps(props);
+  let {landmarkProps} = useLandmark({...props, role: 'region'}, ref);
+  return <article ref={ref} {...landmarkProps} {...styleProps}>{props.children}</article>;
+}
+
+function Search(props) {
+  let ref = useRef();
+  let {styleProps} = useStyleProps(props);
+  let {landmarkProps} = useLandmark({...props, role: 'search'}, ref);
+  return <input type="search" ref={ref} {...landmarkProps} {...styleProps}>{props.children}</input>;
```

Comment:
great to know, thanks for the clarification on that. makes sense because it'd override the role. which I had forgotten was not 'search' already
here's an example of search we can emulate https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/search_role

Reply:
Good catch, pushed a fix